File: core/src/main/java/com/uiptv/model/VodWatchState.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class VodWatchState extends BaseJson {
    private String dbId;
    private String accountId;
    private String categoryId;
    private String vodId;
    private String vodName;
    private String vodCmd;
    private String vodLogo;
    private long updatedAt;
}

File: server/src/main/kotlin/com/uiptv/model/VodWatchState.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

data class VodWatchState @JvmOverloads constructor(
    var dbId: String? = null,
    var accountId: String? = null,
    var categoryId: String? = null,
    var vodId: String? = null,
    var vodName: String? = null,
    var vodCmd: String? = null,
    var vodLogo: String? = null,
    var updatedAt: Long = 0
) : BaseJson()
